src/pages/balances/nfts.tsx

```diff
@@ -23,11 +23,17 @@ const NftApps = memo(function NftApps(): ReactElement | null {
       </Typography>
 
       <Grid container spacing={3}>
-        {nftApps?.map((nftSafeApp) => (
-          <Grid item lg={12} md={4} xs={6} key={nftSafeApp.id}>
-            <SafeAppCard safeApp={nftSafeApp} />
+        {nftApps ? (
+          nftApps.map((nftSafeApp) => (
+            <Grid item lg={12} md={4} xs={6} key={nftSafeApp.id}>
+              <SafeAppCard safeApp={nftSafeApp} />
+            </Grid>
+          ))
+        ) : (
+          <Grid item lg={12} md={4} xs={6}>
+            <Skeleton variant="rounded" height="245px" />
```

Comment:
Knowing that we have 2 `NftApp` in mainnet I'd place 2 skeletons. Your call though

Reply:
I went with 1 because it aligns perfectly with 3 skeleton rows in the NFTs table. Also it's actually 3 apps that we have on prod.